hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -50,25 +72,28 @@ <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackatho
 
                     {% if not rsvp_passed %}
                         <div style="display: flex; width: auto">
+                            {% if status == "Cannot Attend (Declined)" or status == "Accepted, awaiting RSVP" %}
                             <a
                                 href="{{ url("registration:rsvp", kwargs={"rsvp": "yes"}) }}"
                                 class="btn-small waves-effect waves-light colorBtn"
+                                style="margin-right: 30px;"
                             >
                                 Will Attend (Accept)
                             </a>
-
+                            {% endif %}
+                            {% if status == "Will Attend (Accepted)" or status == "Accepted, awaiting RSVP" %}
                             <a
                                 href="{{ url("registration:rsvp", kwargs={"rsvp": "no"})}}"
                                 class="btn-small waves-effect waves-light redBtn"
-                                style="margin-left: 30px;"
                             >
                                 Cannot Attend (Decline)
                             </a>
+                            {% endif %}
                         </div>
 
-                    {% elif rsvp_passed and  application.rsvp == True %}
+                    {% elif rsvp_passed and application.rsvp == True %}
```

Comment:
Since I'm noticing it now apparently, can you make these two `application.rsvp is True` and False accordingly instead of with `==`?